silver/models/payment_processors/base.py

```diff
@@ -13,6 +15,36 @@ def setup(self, data):
 
         raise NotImplementedError
 
+    def get_view(self, transaction, request, **kwargs):
+        kwargs.update({
+            'form': self.get_form(transaction, request),
+            'template': self.get_template(transaction),
+            'transaction': transaction
+        })
+        return self.transaction_view_class.as_view(**kwargs)
+
+    def get_form(self, transaction, request):
+        form = None
+        if self.form_class:
+            form = self.form_class(payment_method=transaction.payment_method,
+                                   transaction=transaction, request=request)
+
+        return form
+
+    def get_template(self, transaction):
+        template = select_template([
```

Comment:
Why move the template selection logic into the "model"? AFAICS Django itself leaves this to the view/s